best-practices-zarr-ndarray.md

```diff
@@ -0,0 +1,536 @@
+# STAC Zarr and N-Dimensional Array Best Practices
```

Comment:
Does it deserve its own markdown file, or should it be somehow merged with "assets and links" best practices?